src/dos/dos.cpp

```diff
@@ -1255,6 +1257,29 @@ class DOS:public Module_base{
 		dos.version.minor=0;
 		dos.direct_output=false;
 		dos.internal_output=false;
+
+		const Section_prop* section = static_cast<Section_prop*>(configuration);
+		const std::string lfn=section->Get_string("lfn");
+		if (lfn=="true") enablelfn=1;
+		else if (lfn=="false") enablelfn=0;
+		else if (lfn=="autostart") enablelfn=-2;
+		else enablelfn=-1;
```

Comment:
`enablelfn` implies an action, ie: _"Please enable LFN"_. Also, seeing it's potential values: -2, -1, 0, 1 don't offer any indication of what's intended.

Suggest moving those values to an enum class, say `LFNState`, and renaming `enablelfn` to indicate it holds state too. `lfn_state` or maybe `lfn_status`, if it can change through the lifetime of the program.

``` c++
if (lfn_state == LFNState::Disabled)
    ...
 else if (lfn_state == LFNState::AutoStart)
    ...
```

::Enabled`,

Reply:
@kcgen For some reason I could not get the enum class to compile with MinGW64 (although it does appear to compile with Visual Studio). However, I have already renamed the variable "enablelfn" to "lfn_state" as you suggested.

Reply:
I have already added full descriptions as comments for all the possible values of lfn_state.